tests/index.js

```diff
@@ -0,0 +1,27 @@
+const test  = require('tape')
+const tapSpec = require('tap-spec')
+// You may need to set another const for relevant JS files
+
+test ('initial', function (t) {
+  t.plan(1)
+  t.equal('a', 'a')
+})
+
+
+test('tell us what you are testing', function (t) {
+  var whatYoureTesting = [] // whatever value you want to test
+  var expected = expectedResult
+  var actual = nowTest(whatYoureTesting) //nowTest would be in your JS file
+  t.equal (actual, expected)
+  t.end()
+})
+
+// If you forgot, this is how a test can look:
```

Comment:
I don't think we need this commented section since you've already included a sample.